<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="toolTip">
    <string>Новый текстовый документ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="toolTip">
    <string>Сохранить файл как</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit.png" resource="texteditor.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выйти из текстового редактора</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="texteditor.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="toolTip">
    <string>Справка по текстовому редактору</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="texteditor.qrc"/>
 </resources>
 <connections/>
</ui>
